package.json

```diff
@@ -30,9 +30,8 @@
     "lodash.clonedeep": "^4.5.0",
     "lodash.isequal": "^4.4.0",
     "lodash.throttle": "^4.1.1",
-    "mapbox-gl": "^0.33.0",
-    "mapbox-gl-inspect": "^1.2.2",
-    "mapbox-gl-style-spec": "^8.11.0",
+    "mapbox-gl": "^0.34.0",
+    "mapbox-gl-inspect": "lukasmartinelli/mapbox-gl-inspect#7108dab80f",
```

Comment:
This points at the git commit because the latest version of mapbox-gl-inspect hasn't been released to npm yet.